src/components/LessonsPIA/LessonsPIA.js

```diff
@@ -4,13 +4,13 @@ import "./LessonsPIA.css";
 
 export default class LessonsPIA extends Component {
```

Comment:
Sarebbe ottimo se riuscissimo ad usare il `TabSelector` direttamente in questa PR, però per ora può bastare anche così